File: app/src/main/kotlin/com/mattdixon/jobjar/util/DurationFormat.kt
package com.mattdixon.jobjar.util

fun formatMinutes(minutes: Int): String {
    if (minutes <= 0) return "0 min"
    val hours = minutes / 60
    val mins = minutes % 60
    return when {
        hours == 0 -> "$mins min"
        mins == 0 -> "$hours hr"
        else -> "${hours}h ${mins}m"
    }
}

File: ios/JobJar/JobJar/Util/DurationFormat.swift
import Foundation

func formatMinutes(_ minutes: Int) -> String {
    guard minutes > 0 else { return "0 min" }
    let hours = minutes / 60
    let mins = minutes % 60
    if hours == 0 { return "\(mins) min" }
    if mins == 0 { return "\(hours) hr" }
    return "\(hours)h \(mins)m"
}
